tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -44,6 +44,14 @@ parameters:
   type: boolean
   default: false
 
+# Id of the pipeline run that contains the artifacts to download.
```

Comment:
would be good to reference https://github.com/microsoft/azure-pipelines-tasks/issues/13518 here IMO, in the off-chance there's any update to it (either in the form of accepting the behavior as a bug or linking documentation that clarifies what triggering pipeline behavior does)

Reply:
Added